js/Source/FlatUI.Select/FlatUI.Select.Filter.js

```diff
@@ -0,0 +1,90 @@
+/*
+---
+
+name: FlatUI.Select.Filter
+
+description: Filter input for FlatUI Select components.
+
+requires:
+ - FlatUI.Select
+ - Form.Filter
+
+
+provides: [FlatUI.Select.Filter]
+
+...
+*/
+
+
+FlatUI.Select.Filter = new Class({
+
+  Extends: FlatUI.Select,
+
+  build: function(){
+    this.parent.apply(this, arguments);
+    // build the DOM elements for our filter, inject them into the list UI
+    this.filterLi = new Element('li.filter').inject(this.list, 'top');
+    this.filterInput = new Element('input.form-control[placeholder=search]');
+    new Element('div.form-group.no-margin.padding-small').adopt(
```

Comment:
hmm. not sure about this class inclusion. i think it should either be explicit styling, or a variable that's passed in.